src/App.tsx

```diff
@@ -8,52 +8,38 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
-import { Counter } from './features/counter/Counter';
+import { useAppDispatch, useAppSelector } from './app/hooks';
 
-export const App: React.FC = () => {
-  const [posts, setPosts] = useState<Post[]>([]);
-  const [loaded, setLoaded] = useState(false);
-  const [hasError, setError] = useState(false);
+import * as postsActions from './features/posts';
+import * as selectedPostActions from './features/selectedPost';
 
-  const [author, setAuthor] = useState<User | null>(null);
-  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+export const App: React.FC = () => {
+  // const [selectedPost, setSelectedPost] = useState<Post | null>(null);
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)